github/resource_github_organization_settings.go

```diff
@@ -0,0 +1,373 @@
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resouceGithubOrganizationSettings() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubOrganizationSettingsCreateOrUpdate,
+		Read:   resourceGithubOrganizationSettingsRead,
+		Update: resourceGithubOrganizationSettingsCreateOrUpdate,
+		Delete: resourceGithubOrganizationSettingsDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"billing_email": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"company": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"twitter_username": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"location": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"has_organization_projects": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"has_repository_projects": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"default_repository_permission": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "read",
+				ValidateFunc: validation.StringInSlice([]string{"read", "write", "admin", "none"}, false),
+			},
+			"members_can_create_repositories": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_create_internal_repositories": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Setting to true allows organization members to create internal repositories. Only available to Enterprise Organizations.",
+			},
+			"members_can_create_private_repositories": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_create_public_repositories": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_create_pages": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_create_public_pages": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_create_private_pages": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"members_can_fork_private_repositories": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"web_commit_signoff_required": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"blog": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"advanced_security_enabled_for_new_repositories": {
```

Comment:
Nice work making changes to the [upstream library](https://github.com/google/go-github/pull/2471) and following through with them here!